core/server/master/src/main/java/alluxio/master/block/meta/MasterWorkerInfo.java

```diff
@@ -501,21 +499,16 @@ public boolean hasLostStorage() {
   @Override
   // TODO(jiacheng): Read lock on the conversion
   public String toString() {
-    Collection<Long> blocks = mBlocks;
-    String blockFieldName = "blocks";
-    // We truncate the list of block IDs to print, unless it is for DEBUG logs
-    if (!LOG.isDebugEnabled() && mBlocks.size() > BLOCK_SIZE_LIMIT) {
-      blockFieldName = "blocks-truncated";
-      blocks = mBlocks.stream().limit(BLOCK_SIZE_LIMIT).collect(Collectors.toList());
-    }
     return MoreObjects.toStringHelper(this)
         .add("id", mMeta.mId)
         .add("workerAddress", mMeta.mWorkerAddress)
         .add("capacityBytes", mUsage.mCapacityBytes)
         .add("usedBytes", mUsage.mUsedBytes)
         .add("lastUpdatedTimeMs", mLastUpdatedTimeMs.get())
         .add("blockCount", mBlocks.size())
-        .add(blockFieldName, blocks)
+        // We truncate the list of block IDs to print, unless it is for DEBUG logs
+        .add("blocks", LOG.isDebugEnabled() ? mBlocks :
+            CommonUtils.summarizeCollection(mBlocks, BLOCK_SIZE_LIMIT))
```

Comment:
Default  `BLOCK_SIZE_LIMIT` is 100, is that still too spammy?  IMHO, i really don't see the difference between 10 entries vs 100 entries..  if we are hoping to see a particular block id in the 100 entries, we will likely be out of luck, considering we tend to lose blocks in batches (e.g lost worker)

Reply:
I agree. I think it serves very little meaning to show 10 entries.
I will simply remove the blocks here but just showing the number of blocks